trusted-committer-article.md

```diff
@@ -0,0 +1,296 @@
+  roles in an InnerSource project so that everyone has the same notion of 
+  the role and is aware what can be expected from s. o. with that role.
+- The "Trusted" in TC conveys that they are trusted (and thus empowered) by both
+  her management and her community to do the job. By fostering openness and
+  transparency, the TC builds trust in the process and the product being built.
+
+## Ensuring Product Quality
+
+- define quality goals
+    - many facets: conceptual integrity, cleanliness of the code, test coverage
+      and other goals
+- ensure that decisions regarding quality are followed through
+- own tech related decisions 
+    - doesn't mean the TC does all the work or mandates what others are doing
+    - TC just needs to be sure the work is done in the end
+    - even if decisions are not made by her, needs to advocate and, if
+      necessary defend them them outside the community.
+- act as stakeholder of the SW/Architecture towards rest of org
+- set an example
+- make sure community has what it needs to produce quality sw
+    - communicated quality goals/standards
+    - empowerment by management (e. g. for necessary refactorings)
+    - the right tools
+    - prioritize quality and release dates
+- ensure quality at strategic level (longevity of the SW)
+- review and accept contributions
+
+## Keeping Things Running Smoothly
+
+- set, enact and communicate contribution guidelines 
+- set guidelines for how to communicate in PRs
+- reviewing contributions
+    - make sure the contribution process uncovers problems, rather than be the
+      cause of them.
+- documenting discovered and yet undocumented requirements 
+    - When an undocumented requirement, process or limitation is discovered, TC
+      is responsible for making sure it is documented accurately and in a way
+      that it is understandable for contributors and POs alike.
+- define the right process for the community
+- adapt existing processes, if necessary
+- gather support for necessary change in how the community operates
+
+## Keeping The Community Healthy
+
+- TCs must strike a healthy balance between effort spent on ensuring product
+  quality and growing a healthy community. 
+- defining/enacting a code of conduct
+- set an example for community members in terms of conduct and especially
+  communication
+- Health also means that there is a critical mass of contributors.
+    - Virtuous cycle: Good mentoring in PRs -> higher level of trust from
+      contributors -> more people being willing to contribute -> more Pull
+      Requests
+- be aware of the communities needs
+    - keep in touch with the community
+- advocate the communities interests
+- Examples:
+    - Treat any contribution as a gift or as a donation. 
+    - I always prioritized onboarding and mentoring over reaching communicated
+      release dates, unless there was a really good reason to meet the date.
+    - Laude excellent and first contributions
```

Comment:
There is [a pattern](https://github.com/paypal/InnerSourcePatterns/blob/master/praise-participants.md) for this.